rollup.config.js

```diff
@@ -1,17 +1,43 @@
+import resolve from "@rollup/plugin-node-resolve";
 import commonjs from "@rollup/plugin-commonjs";
+import replace from "@rollup/plugin-replace";
 import { terser } from "rollup-plugin-terser";
 
-function createConfig(format) {
+function createConfig(format, resolved = false) {
   const dir = format === "module" ? "esm" : format;
+
   return {
-    input: require.resolve("autopublish-template"),
+    input: `src/index.js`,
     output: {
-      file: `${dir}/index.js`,
+      file: `${dir}/index${resolved ? ".resolved" : ""}.js`,
```

Comment:
Just preference, but I prefer naming the output bundles by their package names. Also I like adding `.min.js` for minified bundles, which is something the official package also does

```suggestion
      file: `${dir}/styled-components${resolved ? ".resolved" : ""}.min.js`,
```

Note that the official package also does this: https://unpkg.com/browse/styled-components@5.1.1/dist/styled-components.min.js

Reply:
Sure, I could get behind that as well. Would this be a preference we should try to encourage in the new repo instructions? Or should we recommend following whatever naming convention is preferred by the upstream lib?

Reply:
I think encouraging people to match the package's naming convention for its UMD bundles is the best thing to encourage probably. Not every package has a UMD bundle, though, so then we'd want defaults